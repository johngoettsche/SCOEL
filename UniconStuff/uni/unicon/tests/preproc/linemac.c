#define n 100
#line n
xxx
#define echo2(a,b) a b
#line echo2(1, "one.h")
yyy
#define fname "two.h"
#line n fname
zzz
