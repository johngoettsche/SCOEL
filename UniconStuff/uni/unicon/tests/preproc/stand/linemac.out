#line 99 "linemac.c"

xxx

#line 1 "one.h"
yyy

#line 99 "two.h"

zzz
